docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.1'
+
+services:
+  mysql:
+    image: mysql:5.7
```

Comment:
I'm confused, the above commands in the Makefile did not seem to take advantage of docker-compose did they (for mysql)?